### ISSUE TITLE
Exploration of gift economics

### ISSUE THREAD

**Issue by orthecreedence** [basisproject/tracker#129]
I've had been having many discussions regarding gift economics with a fellow reddit user (see below).

I want to start seriously exploring how this might work. Currently, (as of 2021/03) Basis is set up as a system with wages paid in what amounts to labor vouchers. The primary economy itself *is* effectively a cost-tracking gift economy, but with signals determined through what's effectively a moneyed supply/demand market. So the economy is somewhat shielded by the decoupling of cost and price, but only somewhat, as supply and demand will still drive costs of products and ultimately labor. Obviously a lot of mechanisms change when you remove profit from the equation and [pay for everyone's living expenses](https://github.com/basisproject/tracker/issues/79) but it still smells a lot like a market.

So I'm making this issue to start exploring the idea of a value system not based on money/credits but some form of individual perception of merit.

The general idea is that a person has some amount of buying power. This buying power is not a single value, but is the buying power they have in relation to another person. In other words, if you save someone's cat, your buying power with them might be much higher than your buying power with someone who you bullied in high school. So the cost of a thing is a mixture of its observed production costs and the relationship between the producer/provider and consumer.

Dunbar's number effectively makes this impossible for a global economy, so the interesting and difficult part of this is exploring how to offload the idea of familiarity that would normally be limited to the real social connections you have to some form of electronic tracking. In other words, you could form an instant buying power with someone you've never met, based on their observed activities or relationships within the system.

If there is a fine line to walk between a perfect utopia and a surveillance hellhole, we've been plumetting into hellhole since I opened this issue. So there have to be some tradeoffs to preserve privacy that don't destroy the entire mechanism of the gift economy.

- What aspects *should* we track about products? 
  - We can just track resources and labor (hours), but that's not complete. If a disabled person orders a new wheelchair, it makes sense that anyone who observed that transaction would see it as a need. If an able-bodied person orders one, the cost might be higher because they don't need it. So if the wheelchair is `(N hours labor, X kg steel, Y kg rubber)` then observations about the object itself cannot be systematized.
  - We can track down to the product level itself, but this is an insanely costly undertaking. Now you're teaching a computer *what* a wheelchair is and how it relates to a disabled person and how it relates to an able bodied person. Quantifying objects is difficult enough, but doing their relationships to different people is astronomically complex. I won't say impossible, but I certainly can't figure that shit out.
  - Can we subjectively determine a person's *need* in relation to a product without having expert domain of that person or that object? Perhaps simply by looking at their occupation and the known tools/inputs they need to do their job? To a carpenter, a new hammer might be cheap. To a race car driver, a lambo might be cheap, but only if we know her last lambo was destroyed in a race and she needs a new one.
- How do we reconcile privacy with observed knowledge?
  - In Basis, consumer transactions are private. In the gift economy, some measure of privacy is lost. What things are moved into the public sphere, and what are kept private? I always come back to the classic example: a 10-piece graduated dildo set. How many people would order this if they had a reasonable expectation of privacy? How many of those people would still order it if the purchase was shown to anybody transacting with them? (I would still order it)
  - Can we use ZKPs for some part of this or all? I need to do more research on what their capabilities are in terms of codifying data (as opposed to just numbers).
- The big question: how do gift economics interface with capitalism? With the credit system, there's a bunch of song and dance, but it at least defines the mechanism. With gift economics, how do people and companies buy/sell things from/to the market? There would need to be some translation system (like there is with credits) but it becomes much more ...weird.

I have more concerns but it's late and I need sleep. Will edit this as more things flow into my head.

---

Previous inspirational discussions with shapeshifter83 (seriously, anybody with an interest in gift economics beyond "everything should, like, be free and people will, like, just take what they need you know maan???" should give these a read):

- https://www.reddit.com/r/basisproject/comments/lnnmsf/credit_currency_vs_money_currency_system_is_the/
- https://www.reddit.com/r/CapitalismVSocialism/comments/kfnj9r/socialists_why_are_you_all_not_rioting_against/ggabgra/
- There are others but I'm not digging through my reddit comment history for now. The above discussions are fairly complete.

**Comment by shapeshifter83** [comment]
The user often known as "shapeshifter83" here. I was recently working on this again from a separate angle and noticed that Valueflows listed Basis Project as a user of their system, which tells me we're still working on the same basic big problem. That brought me to here, at long last.

I do still think gift economics is the only possible direction any sort of post-capitalism economy could go, because it's the only arrangement that I can see which produces an incentive structure that self-corrects to prevent the regrowth of exploitation. The diminishing returns effect of power growth in gift economics is critical to stability.

It's well-established that gift economics has superior economic calculation in information-rich environments (which includes human interpretive capability), so the implementation of a scaled gift economy is really about creating or emulating that richness - or at least the strictly necessary aspects - between otherwise information-deficient human relationships.

How to go about this at a technical level? Even after all these years, I'm still not qualified to answer that, but it's clear the solution is close. It's going to involve things like what Valueflows is doing, it's going to involve proof-of-personhood tech, it's going to involve distributed ledgers like Holochain. It's going to involve significant AI detection/event-interpretation capabilities, but economic theory strongly suggests AI should be kept *out* of value-assignment or economic decision-making.

**Comment by orthecreedence** [comment]
Hey, again =].

I'm really glad you're still at it. Checking in 4+ years after creating this issue, I have to admit I'm still stumped. Granted, I've had some other stuff going on the past few years and have shifted focus away from this project so it's not like I'm up until 3am every night thinking about it. But I do still think about it, and plan on picking it up as soon as the life dust settles.

There's some weird mix of tech, culture, and theory that allows gift economics to work, but doing it without turning into a Black Mirror episode is my real concern. The interesting thing is, I believe that because Basis decouples cost-tracking from price calculations (or at least, *allows* decoupling them) it could still very much support different value systems, gift economics included.

> I do still think gift economics is the only possible direction any sort of post-capitalism economy could go, because it's the only arrangement that I can see which produces an incentive structure that self-corrects to prevent the regrowth of exploitation.

I don't fully disagree. Putting aside the weirdness of having some "social score" I do think having our purchasing power limited by the perception of others would certainly create many more favorable outcomes than simply using money. I think viewing exploitation as binary might be a trap...in other words, a moneyed private property corporatist system might have the *most* exploitation, and a gift economy might have the *least* exploitation, but there are spaces in the middle as well, and what other parameters besides exploitation get adjusted for and what are the outcomes of those?

I'm asking genuinely. If we turn exploitation close to zero, what other parameters are adjusted and how? Overall happiness? Ecology? Productive efficiency/teamwork? Diversity (I mean that expansively, not just *skin color* et al)?

> How to go about this at a technical level?

Yeah, wow. How do the observations of the outside world about a person become calculable events? How much trust do we put into those observations? How are those observations gathered and incorporated? How do we override those observations if we *know in our hearts* something to be true/false that the rest of the world disagrees on? Someone somewhere out there truly believes Trump isn't an Epstein-files diddler...how do they incorporate that knowledge into the algorithm? Or not even: yes, he's a diddler, but maybe that persons *likes* diddlers and wants them to succeed. How many of these parameters are there? How many things go into "I like this person?" And how are those incorporated? Maybe it gets simplified: acts that are perceived as "selfish" vs acts that are perceived as "selfless." But even if you reduce it down to "feels good" and "feels bad" you butt up against the social score problem. We can say "we don't release scoring algorithms that let people go below 0" or something, but that doesn't change the fact that the data exists in this network and can be acted upon. VC-backed companies do it all the time: "give us your data in exchange for a cylinder that tells you the weather so you don't have to get off your pimply ass and look outside the window, pinky swear we won't share your data!" Then they collect a bunch of data and turn around and sell it to the nearest scum-sucking ad company. I guess my point is, the data can become a weapon. And if it *can* become a weapon, it *will* become a weapon.

But I'm still baffled as to a) how this could be done in general, and b) how it could be done in a way that doesn't at best cause fatigue by people worrying about others' perception of them day after day or at worst create a new caste system where the popular take all and viciously demote into obscurity anybody who dares challenge them (but do it with a smile so they keep their standing).

In a world where perception controls everything, the ones who can master control of perception become the exploiters. They become the billionaire class. I guess for me, the result of this line of thinking is the question "is there really a way to truly limit exploitative behavior in humanity?" I don't know. You seem to believe gift economics is the solution, but I'm wondering if vectors for power and control simply shift.

One thing does come to mind: is there a way to only operate on economic observations? Forget all the public perception stuff, social scoring, etc etc and just focus on who has contributed X and who has been recompensed Y. "I worked 50 hours as a schoolteacher last week" and "I spent 2 hours making macaroni art the last month" and "I just performed heart surgery" and "I just got four bags of groceries for my family" and "I just got my third lambo." These observations would be rooted in the same system that drives the economy and would essentially be very high-trust and to some extent very low risk (aside from having to make consumer purchases visible to some extent). You wouldn't know if the schoolteacher is *nice* or if the surgeon was drinking before the operation but you could at least to some extent be able to make a judgement on the inputs vs the outputs.

Interestingly, lets say the above was true: we only operate on observed economic information. There might have to be some consensus within companies that serve consumers on which observations are favorable and which are not. If one cashier doesn't like doctors and won't let the surgeon have a sandwich, but the cashier who loves doctors starts her shift in 15 minutes and would gladly make the surgeon TEN sandwiches, that might cause confusion. It might be that when interacting with some larger group, the group itself has agreed upon parameters that would/could override the individual worker (or average the two or something).

> economic theory strongly suggests AI should be kept out of value-assignment or economic decision-making

What's the basis behind this? I'm asking genuinely. In our earlier conversations, you seemed strongly in favor of AI helping economic calculations. Now that we're kind of nearing LLM-in-everyone's-pocket territory, are you saying this angle isn't as viable as previously thought? Or is there a certain piece that AI *would* handle but only as much as it informs some larger calculation?

**Comment by shapeshifter83** [comment]
> I believe that because Basis decouples cost-tracking from price calculations (or at least, _allows_ decoupling them) it could still very much support different value systems, gift economics included.

The nuts and bolts of what Basis does is still largely opaque to me, but these tracked costs - you're tracking them by "units of material", right? That's what I seem to recall. If you're using Valueflows and assuming Valueflows is as fleshed-out as it's marketed as being, you've already got a broad swath of "Resources, Events, and Agents (REA)" with which we can begin to allow algorithmic-but-assisted assignment of either per-user cardinal values (less than ideal, those figures would need storing/transmitting, and it'll need to be transformed to ordinal at the end-user anyway) or find some way to truly determine cardinal-free per-user ordinal values between those "Resources, Events, and Agents" and any (all?) alternative "Resources, Events, and Agents" (this will involve mathematical logic that is probably not terribly advanced but is still beyond me - like that cat said, I'm not really a math guy). I also think we're going to need more than just the Valueflows REA vocab, because context matters (ex: acquisition of a wheelchair by a double amputee is generally considered less consumptive than acquisition of a wheelchair by an annoying YouTuber trying to do dumb stunts, and as such, "costs" are different given contexts, even from the perspective of the same node/observer). We can tack on "Context Tags" to REA so we can call the whole thing "REACT". Got lucky with that one.

> a moneyed private property corporatist system might have the _most_ exploitation, and a gift economy might have the _least_ exploitation, but there are spaces in the middle as well, and what other parameters besides exploitation get adjusted for and what are the outcomes of those?
> 
> I'm asking genuinely. If we turn exploitation close to zero, what other parameters are adjusted and how? Overall happiness? Ecology? Productive efficiency/teamwork? Diversity (I mean that expansively, not just _skin color_ et al)?

The big difference between a gift economy and any other type of economy is the inherent natures of the incentive structure involved. In any system where purchasing power is tracked objectively, whether an objectively-existing medium like gold or paper, or simply numerical objectivity without physical existence, like numbers in a database, the incentive is always to "increase number", or "acquire more of thing". Monkey like bigger number. A really slick system might be able to tune these objective measures to end up _really close_ to alignment with merit, but it's never going to be perfect, because the simple existence of a single power value for an economic actor is the origination point of the entire concept of externalities - misalignment of power with perceived merit - because merit is _always_ subjective and therefore cannot ever match any objective gauge. And _externalities bad_, because humans are very intolerant to them. Like every murder and every war and most crime and every oppressive government and every vote in favor of taxation and every rash action by an envious human, and heck, most simple arguments between folks - it's all externality resolution. Humans really, really, really don't tolerate unresolved externalities.

Gift economics is the (only) system that leaves power subjective, which changes the game entirely. Now power can _actually_ match merit. Now we've removed the primary driver of a massive amount of human effort, time, thought, and action that was expended toward externality-resolution. It can be redirected toward more productivity, or more happiness, which is generally the same thing at the end of the day (the goal of productivity is ultimately happiness anyway).

So, what parameters would be adjusted in a transition from money (more appropriately referred to as balanced reciprocity to catch all forms) to gift economics (generalized reciprocity)? All of them. There would be almost no aspect of human life unaffected.

> How do the observations of the outside world about a person become calculable events?

Via ordinal rankings. We can determine a person's preferences in two ways: by observing their selected actions within contexts and acknowledging the actions _not_ taken, and by querying the person directly about ordinal preferences.

> How much trust do we put into those observations?

Well, it's not going to be perfect. If we use AI to detect, identify, and classify human action, there's going to be errors. But the question is, will the error rate prevent the overall system from outperforming money as the preferred, selected mode of exchange? I think it would be quite tolerant to even a high error rate, because if the error involves something high-value, humans will be inclined to manually correct it, and if it involves something low value, the damage is minimal. And I think overall, the ability for generalized reciprocity to be perceived as vastly preferential to balanced reciprocity is quite plausible - remember, we humans _already_ use generalized reciprocity for the greater _quantity_ of human interaction everyday (just not for high risk or high value stuff right now). Exchange without money is not and will not be as foreign a concept as it may currently appear. Once the larger, overarching effects - call them "meta-effects" - of living in a world without balanced reciprocity become clear, such as the widespread axing of oppressive systems and the end of annoying marketing and the end of homelessness, etc etc, the system will only be even more preferred.

> How are those observations gathered and incorporated?

Should be able to get AI to detect and classify in REACT format, and input into database. Dunno what the actual AI models are for detection in video, but I know in images there's the entire "Yolo" series of detection models that can detect a vast range of "things" in images - I use them in stable diffusion quite often. And there can be other, more human, input sources, obviously.

> How do we override those observations if we _know in our hearts_ something to be true/false that the rest of the world disagrees on?

"Trust" in distributed systems is a whole topic in itself. Holochain has some answers for this, but largely it's going to come from the economic system itself. Being net-productive and well-liked will equal trust, so the more of this - meritorious - a person is, the more influence they can have on what "truth" is within the core database systems. The incentive/disincentive structure of the system will provide some security from maliciousness here too - trying to scam meritoriousness and getting caught would have a greater penalty than any potential gain would have been.

> Someone somewhere out there truly believes Trump isn't an Epstein-files diddler...how do they incorporate that knowledge into the algorithm? Or not even: yes, he's a diddler, but maybe that persons _likes_ diddlers and wants them to succeed.

Well, if the majority of productive humanity thinks this, then it's going to end up being treated as fact and good. Part of making this work will be to accept that morality, like every dang other thing in humanity, is subjective. We have to trust in natural human selfishness to combat this. Yes, you heard that right. Humans _evolved_ their various - but usually eerily similar - senses of morality because adherence to those moralities affected survival rate and happiness. Moral positions exist to throw in wedges and affect productivity/consumption and action in general. And since "diddlers" are widely shunned already, despite our current trash incentive structure, I'm confident that an improved incentive structure will not suddenly shift everyone's evolutionary tendencies away from certain largely agreed-upon moralities. But you should be prepared to accept being a moral minority for some parts of your own personal philosophy. Because it's likely true, and any outsize imposition of any minority positions you might have equals externalities.

> How many of these parameters are there? 

The legit answer is probably: the same number as the number produced by this equation: (nouns x verbs x adjectives)³ .

> How many things go into "I like this person?"

The answer is non-numerical, so how "many" things is unanswerable. The only real answer is ordinal - do I prefer this person vs another person or no person?

> And how are those incorporated?

By mathematical logic, or if we're being lazy, assignment of numerical values in the background that are then used to order things and present an ordinal result with the cardinal result hidden.

> Maybe it gets simplified: acts that are perceived as "selfish" vs acts that are perceived as "selfless."

Productive vs consumptive. One raises merit (generally - it will be, of course, variable per-observer), while the other decreases merit (generally).

> But even if you reduce it down to "feels good" and "feels bad" you butt up against the social score problem. We can say "we don't release scoring algorithms that let people go below 0" or something, but that doesn't change the fact that the data exists in this network and can be acted upon. VC-backed companies do it all the time: "give us your data in exchange for a cylinder that tells you the weather so you don't have to get off your pimply ass and look outside the window, pinky swear we won't share your data!" Then they collect a bunch of data and turn around and sell it to the nearest scum-sucking ad company. I guess my point is, the data can become a weapon. And if it _can_ become a weapon, it _will_ become a weapon.

We just need to be able to know _who_ is trying to misuse data, and the system can record data of this event as well, which can - and should - be judged harshly. Providing disincentive for the act in the first place, in addition to the economic justice delivered.

> But I'm still baffled as to a) how this could be done in general, and b) how it could be done in a way that doesn't at best cause fatigue by people worrying about others' perception of them day after day or at worst create a new caste system where the popular take all and viciously demote into obscurity anybody who dares challenge them (but do it with a smile so they keep their standing).

That's why, especially in the early stages where early-contributor, well-meaning folks like you and I are mostly in control of the value-assignment algorithms with little to no competition, we must gear them toward rewarding productivity. Merit via productivity can't really be faked, so these "social cartels" would need _legit_ high productivity in order to carry on with such an activity, if they really wanted to, which I don't think they would, since if they already have legit merit, I don't think they would risk the potentially massive consequences of also trying to farm illegitimate merit. I mean, some otherwise-productive people are morons and will do this, but they'll get what they actually, legitimately deserve.

> In a world where perception controls everything, the ones who can master control of perception become the exploiters. They become the billionaire class. I guess for me, the result of this line of thinking is the question "is there really a way to truly limit exploitative behavior in humanity?" I don't know. You seem to believe gift economics is the solution, but I'm wondering if vectors for power and control simply shift.

_We_ would be that gift economic version of a "billionaire" class. Nobody else is working on this, as far as I can tell. You, me, Foster and Haugen at Valueflows... and they are much older than you and I, and certainly retiring soon, if not nominally retired already. If David Graeber hadn't died he'd be working on this with us, no doubt. Maybe Charles Eisenstein, but he's losing focus as he gets older, I've noticed. We are it Andrew. Us, and whoever we recruit, really.

Eventually we'd gradually lose control as new algorithms come online, until we regressed back toward the mean, at least somewhat - we just have to make sure we align early systems with true merit (the truest merit being productivity) to the best of our abilities and do our best not to inject our own biases. We need to establish an identifiable pattern linking productivity, unbiased truth, and broad unfiltered data dissemination with higher quality of life for everyone. The economics of Misesian economic calculation are already on our side in that regard - that's von Mises exact argument for the superiority of money, due to it's superior calculation (over unassisted systems). If we do one better on the calculation front than von Mises' analysis of money, theoretically it should come with a corresponding further increase in widespread QoL over money.

> One thing does come to mind: is there a way to only operate on economic observations? Forget all the public perception stuff, social scoring, etc etc and just focus on who has contributed X and who has been recompensed Y. "I worked 50 hours as a schoolteacher last week" and "I spent 2 hours making macaroni art the last month" and "I just performed heart surgery" and "I just got four bags of groceries for my family" and "I just got my third lambo." These observations would be rooted in the same system that drives the economy and would essentially be very high-trust and to some extent very low risk (aside from having to make consumer purchases visible to some extent). You wouldn't know if the schoolteacher is _nice_ or if the surgeon was drinking before the operation but you could at least to some extent be able to make a judgement on the inputs vs the outputs.

The problem is that it just wouldn't work to leave out the subjective inter-woven layers of human-to-human value judgements. It wouldn't outperform money. It wouldn't be selected organically as a method of exchange. It's not resolving externalities in this case. You've still got the incentive issue: merit is not directly tapped, instead the objective is to increase "recompense Y". By the easiest and lowest-effort means possible.

In order to overcome money, we have to position our system as selectable over status quo monetary systems. Something needs to drive people toward our system. The primary driver is externality resolution. And externalities are, specifically, a _misalignment of power with perception of merited power_. A great place to start will be ad-rejection. Think: AI on our phones identifies ads and the corresponding product provider. Then, it helps prevent you from buying their product and instead helps you find alternatives that didn't spam you with unskippable ads. People _want_ that ability. It's a resolution of an externality. You're into environmental ecology. I can't think of an example but I'm sure you could. Environmental externalities are another spot where our system could get early popularity because it could provide methods of legitimate externality resolution.
 
> Interestingly, lets say the above was true: we only operate on observed economic information. There might have to be some consensus within companies that serve consumers on which observations are favorable and which are not. If one cashier doesn't like doctors and won't let the surgeon have a sandwich, but the cashier who loves doctors starts her shift in 15 minutes and would gladly make the surgeon TEN sandwiches, that might cause confusion. It might be that when interacting with some larger group, the group itself has agreed upon parameters that would/could override the individual worker (or average the two or something).

Need to make sure all value judgments are human-to-human. If it becomes human-to-group, then that opens the door to misalignments of power with merit, due to free riders in that group, or just a general spectrum of productivity within that group but a flat application of a power assignment.

> > economic theory strongly suggests AI should be kept out of value-assignment or economic decision-making
> 
> What's the basis behind this? I'm asking genuinely. In our earlier conversations, you seemed strongly in favor of AI helping economic calculations. Now that we're kind of nearing LLM-in-everyone's-pocket territory, are you saying this angle isn't as viable as previously thought? Or is there a certain piece that AI _would_ handle but only as much as it informs some larger calculation?

Ah, well, my position has always been that humans need to do any value assignment, but my acceptance of AI for everything around that has certainly opened up in the past couple years.

Bottom line is this: AI is not a human, and therefore cannot accurately assign value judgments in the human economic system. Just as we are not squirrels, and therefore cannot accurately assign value judgments in the squirrel economy. Only a squirrel truly understands the economic value of cheeks full of acorns. Just like only humans truly understand the value of human beauty or water when parched or a pillow when exhausted. AI can help with things, but too much AI injecting itself into human assignments of value will have the same sort of effect of corrosion/limitation of economic data as socialism concepts such as restrictions against capital exchange.

I'm off to work, finished this just in time - I'll check again here more often.

**Comment by orthecreedence** [comment]
> The nuts and bolts of what Basis does is still largely opaque to me, but these tracked costs - you're tracking them by "units of material", right

Kind of, yeah. There are three categories: labor (by occupation, tracking both hours and wages), resources (lumber, iron, oil, etc), and processes (refine oil turns crude oil -> gasoline, jet fuel, etc). Of all of those things, the only categories tracked are ones specifically added to the network's knowledge base, meaning it doesn't track *everything* but rather tracks aggregations of pre-agreed-upon categories. People are free to get as specific as needed, but ultimately it doesn't really make sense to model any resources that don't have known large-scale externalities. So instead of tracking each individual screw in a chair, you might just track the steel content of those screws. If you follow the REA chain back through the network, you might be able to see how many screws were used in the chair, but that number is not stored with the chair as it moves forward in the productive system, only the aggregate labor, resources, and processes the chair or its constituent parts went through to come into creation.

The end goal is to be able to tell, for each product/service, exactly how much labor went into providing it and what *potentially harmful or scarce* resources went into providing it. The final collection of labor, resources, and processes can be converted into a number given some network-defined values for each of the tracked categories, meaning a given product can be reduced to a single number value "cost" which is representative not just of production costs, but known externalities. The same product might have a lower cost today than it does tomorrow if it has a lot of fossil fuel content and people decide to rank fossil fuels as more costly network-wide.

Think of it like a giant disaggregate accounting system that allows self-imposed Pigouvian taxes based on collective knowledge. There's UBI and shit too which is a form of generalized reciprocity ;).

> So, what parameters would be adjusted in a transition from money (more appropriately referred to as balanced reciprocity to catch all forms) to gift economics (generalized reciprocity)? All of them. There would be almost no aspect of human life unaffected.

Yes, for sure, and I guess my curiosity makes me wonder which of those parameters would be adjusted negatively. I've found in life that net-gains tend to be rare or minimal, and instead trade-offs are more likely. What becomes worse as a result of this shift?

> Well, it's not going to be perfect. If we use AI to detect, identify, and classify human action, there's going to be errors. But the question is, will the error rate prevent the overall system from outperforming money as the preferred, selected mode of exchange? I think it would be quite tolerant to even a high error rate, because if the error involves something high-value, humans will be inclined to manually correct it, and if it involves something low value, the damage is minimal.

Thing is, trust is still very much in play. A fancy, infallible AI could actually be a teenager with a hacked version of the app who marks her high school principal as a diddler. There would be very little automated input that could be trusted, and any data that was input manually would be (and should be) heavily scrutinized.

I guess my point is that when distribution becomes a function of perception, money is no longer a number but rather the ability to modify or influence perception. If people are looking to these observations to determine if one person they have never met is preferential over another person, there needs to be some trust in the data, or it will all just be random. This means some enforcement mechanisms to make sure the currency being used is not "counterfeit."

In effect, without building this anti-counterfeiting, we're taking something particularly dangerous (making snap judgements about someone you've never met in order to determine not just how you *feel* about them but whether they are worthy of some resource) and scaling it to an immense level.

I guess my point is, the more I think about it, the more uncomfortable I am with using anything other than the (semi-)objective economic measurements Basis already records. Otherwise, all of this observation data needs to be arbitrated by someone/something. That arbitration has an *incredibly* high cost, especially in a world where people can generate a somewhat photo-realistic video of anyone doing anything.

> The incentive/disincentive structure of the system will provide some security from maliciousness here too - trying to scam meritoriousness and getting caught would have a greater penalty than any potential gain would have been.

Again, caught for what, by who? And how? And finally, how is that observation realized? You can say "only information added by people you personally extend trust to is trusted" and that's actually a great solution, but if you only trust observations made by people you trust, then you're effectively back to Dunbar's number. But I don't see another way.

At a time when perception and truth has been fractured so much, I almost view shielding ourselves and our economic transactions from the perceptions of others as a feature rather than a bug.

> The answer is non-numerical, so how "many" things is unanswerable. The only real answer is ordinal - do I prefer this person vs another person or no person?

Makes sense.

> Productive vs consumptive. One raises merit (generally - it will be, of course, variable per-observer), while the other decreases merit (generally).

Points to favoring economic data, since many of these actions are already tracked. That said, I'm not blind to the fact that separating the productive system from the culture at large has its own problems. But I'm hoping you see why having an open database that tracks *every good or bad thing anybody has ever supposedly or definitely done* gives me pause.

I hope my responses to you over the years haven't given you the impression I'm not really in favor of gift economics. But I'm really uneasy about creating some instantly-searchable database about every person on the planet that includes everything they've done.

> We just need to be able to know _who_ is trying to misuse data, and the system can record data of this event as well, which can - and should - be judged harshly.

Easier said than done! And also another subjective record of wrongdoing, submitted by someone either truthfully or with an axe to grind...who's to say what's real and what isn't? We have courts for that, but litigating every single record (or even just high value ones) would grind things to a halt.

> Merit via productivity can't really be faked, so these "social cartels" would need _legit_ high productivity in order to carry on with such an activity

I'm confused though. How is productivity measured? Aren't we really just measuring if we like someone more than someone else? Where does productivity fit in? Maybe I like him because he's got great hair, or I like her because she looks hot in a tight red dress. Are those people *producing things* for society? Nice hair and great bodies, no doubt, and maybe that's enough, but I wouldn't economically or logically consider someone simply *being attractive* as being productive. Yet, my ape brain doesn't care and wants to just score them high.

But if we are measuring productivity by economic inputs and economic outputs, then it really can just be assigned an ordinal number and we're back to money.

So I guess what's your definition of productivity here that I'm missing? Are you saying the scoring algorithms will separate behaviors based on their perceived contribution to the whole from behaviors that are still favorable but less contributory?

> You've still got the incentive issue: merit is not directly tapped, instead the objective is to increase "recompense Y". By the easiest and lowest-effort means possible.

In a scaled-public-perception gift economics system, isn't there still a lowest-effort incentive? What if it's easier to convince people I'm doing great things rather than just doing great things? Also if you regularly do something really important, but nobody sees or recognizes it besides maybe a handful of people, then there's low incentive. Now not only are you trying to do your work, you're also trying to convince people why it's important, continuously. I referenced exhaustion in my previous comment...it seems like it would be a factor here.

> And externalities are, specifically, a _misalignment of power with perception of merited power_.

We have different definitions =]. I view externalities as harmful unrealized costs at-scale. I believe it's completely possible to build a society with generalized reciprocity that completely ignores externalities (as I define them) and steers the ship into the rocks.

> Think: AI on our phones identifies ads and the corresponding product provider. Then, it helps prevent you from buying their product and instead helps you find alternatives that didn't spam you with unskippable ads. People _want_ that ability. It's a resolution of an externality.

I understand this logic chain, but I'm curious how this would play out. You could punish the creators of the ad, or, more appropriately, the creators of the platform that allowed unskippable ads in the first place. Do you find everyone who built that platform (past and present) and devalue their merit to 0? And I'm guessing it would take many people doing this to actually affect those people meaningfully (I would hope, anyway). Does the system demand a reason for this mass demeriting of people so they know "oh, I can't get a new pair of shoes this week because I worked on that dumb video app six years ago" or is it all a mystery?

> You're into environmental ecology. I can't think of an example but I'm sure you could. Environmental externalities are another spot where our system could get early popularity because it could provide methods of legitimate externality resolution.

I guess you could say "booo I want all oil execs to be tarred and feathered." Or you could demerit anybody who has purchased gasoline in the last year. In Basis you'd vote for the externality-cost on various fossil fuels to increase. Both likely have a similar effect.

> Need to make sure all value judgments are human-to-human. If it becomes human-to-group, then that opens the door to misalignments of power with merit, due to free riders in that group, or just a general spectrum of productivity within that group but a flat application of a power assignment.

I think that would ultimately completely break down the fabric of the modern productive system, at least in its current form. I'm curious how things could operate without economic groups with defined boundaries. In the anarcho-communist sense it might work where all "property" is everyone's, sure (granted there's friction there deciding who can use what and when). But aren't you an ancap? How would the mediation between individuals and a resource owned by a group of people occur if that mediation layer wasn't highly controlled in some aspect? Would it be via association only? "Only people who install this value program can work here." But then, it's simply just a human-to-group judgement anyway, with an external enforcement system. Another aspect: "fundraising." I mentioned before tasks that are incredibly important but most people don't know why. Now imagine that person needs allocation for their task, but people don't understand why. They might need an advocate for that economic activity. They would necessarily need to function as a unit: the worker and the advocate. Both essential roles, done by separate people, for a common task. This reads as a group activity to me, requiring group judgement by others.

I guess part of me is incredibly intrigued by peer-to-peer production, but ultimately I've dismissed it as inefficient given that some defined social structure around shared resources ("owned" or "unowned") makes the use of those resources much more streamlined. My thinking is that groups inevitably form anyway, so why not build for that?

I'm interested to hear your thoughts on this.

> Ah, well, my position has always been that humans need to do any value assignment

Ok, so your thoughts are that AI can help in gathering and interpreting the data, but the final judgement needs to come from a human? That makes sense, but I'm curious where the threshold lies. But that's probably a satellite discussion.
